mysql/tests/it/main.rs

```diff
@@ -13,4 +13,3 @@
 // limitations under the License.
 
 mod r#async;
-mod sync;
```

Comment:
I think the sync test is to check if it can be adapted to the sync client crate (mysql), so it may need to be kept.

In msql-srv, the difference between the two tests is the difference in client crate.

Reply:
I rewrote the sync test to test the sync mysql client.